_posts/2019-11-01-snowplow-r117-biskupin/index.md

```diff
@@ -0,0 +1,151 @@
+
+The latter allows for low-level TLS/SSL configuration exposed by [Lightbend SSL Config](https://lightbend.github.io/ssl-config/index.html).
+​
+For example to start up an ssl-enabled, auto-upgrade server, following config can be used:
+```json
+ssl {
+  enable = true
+  redirect = true
+  port = 443
+}
+```
+However, this configuration will use environment-defined JVM-attached certificates. In order to override the default behaviour and use a custom certificate, the low-level section can be defined as:
+```json
+ssl-config {
+  keyManager = {
+    stores = [
+      {type = "PKCS12", classpath = false, path = ${CERT_FILE}, password = "pass" }
+    ]
+  }
+}
+```
+​
+### Default redirect endpoint disabling
+Another simple, but important security improvement is a possibility to disable the default event submission redirect. This allows users to disable the default endpoint in favour of custom user-defined url. 
+For example, following configuration will only allow redirects for custom-defined `/com.acme/redirect-me` endpoint, whereas the default `/r/tp2` will not be available.
+```json
+enableDefaultRedirect = false
+paths {
+  "/com.acme/redirect-me" = "/r/tp2"
+}
+```
+Note, that for backwards-compatibility, the endpoint is still exposed by default.
+
+## Referer parser refreshment
+
+Referer parser is a library that contains list of possible referrer hosts. We keep maintenance of `0.3.x` branch with backporting new identified hosts into old `0.3.0` version. Several new hosts were identified and added to the [`referer.yml`](https://s3-eu-west-1.amazonaws.com/snowplow-hosted-assets/third-party/referer-parser/referers-latest.yml)
+
+## IPv6 anonymization
+
+This release adds support for anonymization of IPv6 addresses. `anonSegments` parameter is added to enrichment configuration. It describes how many segments should be anonymized. Some examples might be found in our test scenarios in `AnonIpEnrichmentSpec` in `3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/AnonIpEnrichmentSpec.scala`.
+
+The schema has been updated to version `1-0-1`:
+
+```json
+{
+	"schema": "iglu:com.snowplowanalytics.snowplow/anon_ip/jsonschema/1-0-1",
+	"data": {
+		"name": "anon_ip",
+		"vendor": "com.snowplowanalytics.snowplow",
+		"enabled": true,
+		"parameters": {
+			"anonOctets": 1,
+			"anonSegments": 1
+		}
+	}
+}
+```
+
+The IPv6 anonymization changes are backwards compatible. The `anonOctets` param will be read and useed also for IPv6 in case of missing `anonSegments`. To fully use IPv6 anonymization it's better to use the new schema (version `1-0-1`, which has `anonSegments` param). The new code can be used with `1-0-0` schema version also.
+
+## Additional event fingerprint hashing methods
+
+Credits for this contribution goes to [miike](https://github.com/miike).
+
+From this version, besides MD5, new hashing algorithms are supported:
+
+* SHA1
+* SHA256
+* SHA384
+* SHA512
+
```

Comment:
Would be nice to precise that the version of the configuration has been updated.